Dockerfile

```diff
@@ -3,12 +3,15 @@ WORKDIR /usr/src/app/
 
 COPY .babelrc package.json package-lock.json ./
 RUN apk add --no-cache --virtual .gyp python3 make g++ &&\
-    npm install &&\
+    apk add dumb-init &&\
+    npm ci --only=production &&\
     apk del .gyp
 
 COPY ./src/ ./src/
 COPY ./scripts/ ./scripts/
 COPY config.js.docker ./src/config.js
 
+RUN npm run build
```

Comment:
suggestion: Maybe we could have a multi-stage dockerfile, because we only need the `dist` folder and `dumb-init` to run the program, right?

The rest is only needed for the build (`python3`, `make`, `g++`)

Reply:
I think we should do this, but in another PR where we can include some other best practices and updates.